### DISCUSSION TITLE
firmwareのビルドについて

### DISCUSSION THREAD

**Discussion by yummyKB** [Yowkees/keyball#145] [Q&A]
お世話になります。
keyballのfirmwareをいじってみたいのでまずビルドできる環境を作って
取りあえずdefaultのfirmwareをビルドしようとしているのですがうまく
行きません。

How to build の3まではできていると思うのですが4で
Segmentation fault（コアダンプ）と表示されてしまいます。
解決方法があれば教えて欲しいです。

因みに環境は
win11
QMK MSYS
です。

よろしくお願いします。

**Comment by Yowkees** [comment]
ご連絡が遅くなってしまい申し訳ありません。
ちょっと私も見たことがないエラーでアドバイスができません。すみません。
私がWin11環境をまだ導入していませんのでそのあたりの関係でしょうか。。
